Gems/AWSClientAuth/Code/Source/Authorization/AWSCognitoAuthorizationController.cpp

```diff
@@ -65,9 +68,13 @@ namespace AWSClientAuth
         AWSCore::AWSResourceMappingRequestBus::BroadcastResult(
             m_cognitoIdentityPoolId, &AWSCore::AWSResourceMappingRequests::GetResourceNameId, CognitoIdentityPoolIdResourceMappingKey);
 
-        if (m_awsAccountId.empty() || m_cognitoIdentityPoolId.empty())
+        if (m_awsAccountId.empty())
+        {
+            AZ_Printf("AWSCognitoAuthorizationController", "AWS account id not not configured. Proceeding without it.");
```

Comment:
nit: We will log this every time, this may want to either move to AZ_Trace or have log level other than info set. Not sure if it merits log line as this is optional and not required, its more now that the inverse is true ie

"Optional AWS Account provided, will be used for extra validation in calls to Cognito"